server/stream.go

```diff
@@ -4610,6 +4612,11 @@ func (mset *stream) setConsumer(o *consumer) {
 	if o.cfg.FilterSubject != _EMPTY_ {
 		mset.numFilter++
 	}
+	for _, subject := range o.cfg.FilterSubjects {
```

Comment:
Can it be valid after checking config that an entry be == _EMPTY_?

Reply:
if you pass `filterSubjects: []string{"", "two"} it will fail the. validation and return overlapping subjects error.

Reply:
So here subject can not ever equal _EMPTY_ correct?

Reply:
correct, I now treat this as an error (client passing `[]string{"}`